### ISSUE TITLE
Font icons are not shown if --env.aot flag is used during build

### ISSUE THREAD

**Issue by erkanarslan** [NativeScript/nativescript-cli#5174]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.2.2
 - Cross-platform modules: 6.1.2
 - Android Runtime: 6.2.0
 - iOS Runtime: 6.2.0
 - Plugin(s): Unrelated

**Describe the bug**
I use font icons in my app as described in this page: https://docs.nativescript.org/ui/components/icon-fonts

During development I can see the font icons in the app. However, if I build the app with --env.aot flag, icons are not shown in the app. This is my build command:

    tns build ios --bundle --env.aot --release --forDevice --teamId xxxxxxxx

If I remove --env.aot flag, I can see the icons but cannot benefit from performance improvements.

Same is true for Android.

**To Reproduce**
Create a project that uses font icons. Build it with --env.aot flag.

**Expected behavior**
You should see the icons even when you build it with --env.aot

**Sample project**
I can send you my whole project to test it. Just give me a mail address.

**Additional context**
None

**Comment by rosen-vladimirov** [comment]
Hey @erkanarslan ,
We are not aware of issues with the AOT flag and font icons. In fact, some of [our templates]() are using Font Awesome and everything works as expected. You can give it a try by creating a new application on your side:
```
tns create myApp --template tns-template-master-detail-ng
```
If you are still unable to determine what's causing the unexpected behavior on your side, can you please send your full project.

**Comment by erkanarslan** [comment]
I found out that when this occurs. I use fonticon pipe of nativescript-ngx-fonticon library to transform font-awesome icon names to unicode characters in the xml of the interface. Example:

    <action-item icon="font://{{'fa-search' | fonticon}}" class="fas"></action-item>

If I use this pipe in the action-item element and build the application with --env.aot flag, it throws `NativeScript encountered a fatal error: TypeError: null is not an object (evaluating 'img.imageWithRenderingMode')`.

However, if you use the fonticon pipe in a label element or you don't build the project with the --env.aot flag, no error occurs.

Currently, I removed the pipe and used the unicode character directly to make a workaround. Example:

    <action-item icon="font://&#xf067;" class="fas"></action-item>
